### PR TITLE
Modifying the pool name to include the service name for consistent naming in kubernetes and openshift use cases.

### PR DESCRIPTION
**Pull request by parimanur**
…

### CODE REVIEW

**Comment by parimanur** [comment]
build ako

File: internal/nodes/avi_model_evh_nodes.go [hunk 1]
```diff
@@ -629,7 +629,7 @@ func (o *AviObjectGraph) BuildPolicyPGPoolsForEVH(vsNode []*AviEvhVsNode, childN
 		}
 
 		var poolName string
-		poolName = lib.GetEvhVsPoolNPgName(ingName, namespace, host, path.Path)
+		poolName = lib.GetEvhVsPoolNPgName(ingName, namespace, host, path.Path, path.ServiceName)
```
Comment (monotosh-avi) [inline internal/nodes/avi_model_evh_nodes.go:632]: I think now we can remove isIngr from functions arguments in evh

**Review by sudswasavi — DISMISSED** [review]

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Comment by monotosh-avi** [comment]
build ako

**Review by monotosh-avi — APPROVED** [review]

**Review by sudswasavi — APPROVED** [review]

**Pull request merged**